### DISCUSSION TITLE
How to auto-comment when open a PR?

### DISCUSSION THREAD

**Discussion by linrongbin16** [community/community#69455] [Actions] [answered]
### Select Topic Area

Question

### Body

Hi, I'm trying to add a PR template when contributors open new PR.

And the template content is a quite long markdown content:

```markdown
# Regresion test

## Platform

- [ ] windows
- [ ] macOS
- [ ] linux

## Tasks

- [ ] FzfxFiles
  - [ ] Press `CTRL-U`/`CTRL-R` to switch between restricted/unrestricted mode, and the lines count is consistent when press multiple times.
  - [ ] Use `V`/`W`/`P` variants (visual selection, cursor word, yank text).
  - [ ] Press `ESC` to quit, `ENTER` to open file.
- [ ] FzfxLiveGrep
  - [ ] Press `CTRL-U`/`CTRL-R` to switch between restricted/unrestricted mode, and the lines count is consistent when press multiple times.
  - [ ] Use `-w` to match word only, use `-g *.lua` to search only lua files.
  - [ ] Use `V`/`W`/`P` variants (visual selection, cursor word, yank text).
  - [ ] Press `ESC` to quit, `ENTER` to open file.
- [ ] FzfxBuffers
  - [ ] Press `CTRL-D` to delete buffers.
  - [ ] Use `V`/`W`/`P` variants (visual selection, cursor word, yank text).
  - [ ] Press `ESC` to quit, `ENTER` to open file.
- [ ] FzfxGFiles
  - [ ] Press `CTRL-U`/`CTRL-W` to switch between workspace/current folder mode.
  - [ ] Use `V`/`W`/`P` variants (visual selection, cursor word, yank text).
  - [ ] Press `ESC` to quit, `ENTER` to open file.
- [ ] FzfxGCommits
  - [ ] Press `CTRL-U`/`CTRL-A` to switch between git repo commits/current buffer commits.
  - [ ] Use `V`/`W`/`P` variants (visual selection, cursor word, yank text).
  - [ ] Press `ESC` to quit, `ENTER` to copy commit hash.
- [ ] FzfxGBlame
  - [ ] Use `V`/`W`/`P` variants (visual selection, cursor word, yank text).
  - [ ] Press `ESC` to quit, `ENTER` to copy commit hash.
- [ ] FzfxLspDiagnostics
  - [ ] Press `CTRL-U`/`CTRL-W` to switch between workspace/current buffer diagnostics.
  - [ ] Use `V`/`W`/`P` variants (visual selection, cursor word, yank text).
  - [ ] Press `ESC` to quit, `ENTER` to open file.
- [ ] FzfxLspDefinitions, FzfxLspTypeDefinitions, FzfxLspReferences, FzfxLspImplementations
  - [ ] Go to definitions/references (this is the most 2 easiest use case when developing this lua plugin with lua_ls).
  - [ ] Press `ESC` to quit, `ENTER` to open file.
- [ ] FzfxFileExplorer
  - [ ] Press `CTRL-U`/`CTRL-I` to switch between filter/include hidden files mode.
  - [ ] Press `CTRL-L`/`CTRL-H` to cd into folder and cd upper folder.
  - [ ] Press `ESC` to quit, `ENTER` to open file.

```

I searched some auto-comment github actions ([auto comment](https://github.com/marketplace/actions/auto-comment)) in marketplace, but they only provide very simple `messages`, which not work for my long markdown content.

Would you please help me with that?

**Comment by seyLu** [comment, marked as answer]
#### Hi @linrongbin16,

Github supports [pull request templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository). One way to enable it every PR, is to add a PULL_REQUEST_TEMPLATE.md under `.github/` directory -- also make sure it is located on the default branch (usually `main` or `master`).

```bash
.github
  |-- workflows
        |-- ...
  |-- dependabot.yaml
  |-- PULL_REQUEST_TEMPLATE.md
 src
  |-- ...
 .gitignore
```
![image](https://github.com/community/community/assets/98249191/8be8e7cf-1b5c-404c-9070-ba4aae049a7c)
![image](https://github.com/community/community/assets/98249191/915855b0-abd9-41dc-845b-ecbc72ee6e8e)

For some reason, putting pr template under `.github/PULL_REQUEST_TEMPLATE` doesn't work:
- [StackOverflow: Github PR template not showing](https://stackoverflow.com/questions/52139192/github-pull-requests-template-not-showing)

**Reply by linrongbin16** [reply]
thanks so much, I tried many times yesterday and just cannot make it work.

**Reply by nicholaspshaw** [reply]
@seyLu is it possible to make this template apply to only certain filepaths?  Therefore it wouldnt post on every PR but only those relevant i.e. regression tests folder for example
